packages/cdk/lib/stacks/engines/toil-engine-construct.ts

```diff
@@ -0,0 +1,104 @@
+import { Aws } from "aws-cdk-lib";
+import { IVpc } from "aws-cdk-lib/aws-ec2";
+import { FargateTaskDefinition, LogDriver } from "aws-cdk-lib/aws-ecs";
+import { ApiProxy, SecureService } from "../../constructs";
+import { IRole } from "aws-cdk-lib/aws-iam";
+import { createEcrImage, renderServiceWithTaskDefinition } from "../../util";
+import { Bucket } from "aws-cdk-lib/aws-s3";
+import { EngineOptions, ServiceContainer } from "../../types";
+import { LogGroup, ILogGroup } from "aws-cdk-lib/aws-logs";
+import { EngineOutputs, EngineConstruct } from "./engine-construct";
+import { ToilJobRole } from "../../roles/toil-job-role";
+import { ToilEngineRole } from "../../roles/toil-engine-role";
+import { IJobQueue } from "@aws-cdk/aws-batch-alpha";
+import { Construct } from "constructs";
+
+export interface ToilEngineConstructProps extends EngineOptions {
+  /**
+   * AWS Batch JobQueue to use for running workflows.
```

Comment:
Is it used for running workflows or workflow tasks (or both)?

Reply:
This is just for the tasks; I can note that.